### PR TITLE
[ticket/14213] Adding event after users have been added to a group

### PR DESCRIPTION
**Pull request by Wardormeur**
The idea is to be able to map with others CMS by exposing most commons user configuration & changes

[PHPBB-14213](https://tracker.phpbb.com/browse/PHPBB3-14213)

### CODE REVIEW

File: phpBB/includes/functions_user.php [hunk 1]
```diff
@@ -2754,6 +2754,18 @@ function group_user_add($group_id, $user_id_ary = false, $username_ary = false,
...
 		$db->sql_multi_insert(USER_GROUP_TABLE, $sql_ary);
+		/**
+		* Event after users are added to a group
+		*
+		* @event core.group_add_user_after
+		* @var	int		group_id		ID of the group to which users are added
+		* @var	string	group_name		Name of the group
+		* @var	array	user_id_ary		IDs of the users which are added
+		* @var	array	username_ary	names of the users which are added
+		* @since 3.7
```
Comment (marc1706) [inline phpBB/includes/functions_user.php]: Sould be 3.1.7-RC1

File: phpBB/includes/functions_user.php [hunk 2]
```diff
@@ -2754,6 +2754,18 @@ function group_user_add($group_id, $user_id_ary = false, $username_ary = false,
...
+		*
+		* @event core.group_add_user_after
+		* @var	int		group_id		ID of the group to which users are added
+		* @var	string	group_name		Name of the group
+		* @var	array	user_id_ary		IDs of the users which are added
+		* @var	array	username_ary	names of the users which are added
+		* @since 3.1.7-RC1
+		*/
+		$vars = array('group_id', 'group_name', 'user_id_ary', 'username_ary');
+		extract($phpbb_dispatcher->trigger_event('core.group_add_user_after', compact($vars)));
```
Comment (Nicofuma) [inline phpBB/includes/functions_user.php]: It should be done after the transaction commit and even after the cache clear (line 2788), like in the delete event.

Comment (Wardormeur) [inline phpBB/includes/functions_user.php]: Just saw it, will fix, thanks

**Comment by marc1706** [comment]
Thank you for your PR. I have take the liberty to fix the issues reported by travis in a replacement PR.
All your commit messages and with ```PHPBB-14213```. They should however end with ```PHPBB3-14213```. Please note the extra ```3``` after ```PHPBB```. Please try to use this correctly in your next PR.

**Comment by Wardormeur** [comment]
Hey @marc1706, thanks a lot.
I take good note for the PHPBB[3], but mind telling me what was wrong on this PR ?
Thanks :)

**Comment by marc1706** [comment]
Nothing was wrong apart from the commit messages. I just decided to take this over in order to fast track this. As you can see in #4071 your contributions are still preserved.

**Comment by Wardormeur** [comment]
No worries, on the contrary, I'm glad :)
Thanks! :+1:

**Pull request closed without merging**